src/clj/rems/css/styles.clj

```diff
@@ -647,14 +650,21 @@
      (s/descendant :.language-switcher :form)
      :form)
     {:margin-left (u/rem 0.5)}]
-   [:.commands {:text-align "right"
-                :padding "0 1rem"
-                :cursor :auto}]
+   [:div.commands {:cursor :auto
+                   :display :flex
+                   :flex-direction :row
+                   :flex-wrap :wrap
+                   :gap (u/rem 0.5)
+                   :align-items :center
+                   :justify-content :flex-end}]
+   [:td [:div.commands {:justify-content :flex-start}]]
+   [:th.organization {:white-space :normal
+                      :min-width (u/rem 5.5)}]
+   [:th.active {:white-space :normal
+                :min-width (u/rem 5.5)}]
```

Comment:
Is the `white-space: normal` required?

Reply:
Yes, normally `th` is `nowrap` but the word "Organization" is rather long compared to an abbreviated organization name, so we should let it wrap.